strats.md

```diff
@@ -520,6 +656,32 @@ A `comeInJumping` entrance condition represents the need for Samus to be able to
 }
 ```
 
+### Come In Space Jumping
+
+A `comeInSpaceJumping` entrance condition indicates that Samus must come in with a Space Jump through the bottom of the doorway, applicable to horizontal transitions. It has the following properties:
+
+* _speedBooster_: If true, then Speed Booster must be used while gaining run speed. If false, then Speed Booster must not be used. If "any", then Speed Booster may or may not be used.
+* _minTiles_: The minimum horizontal speed that will satisfy the condition, measured in effective runway tiles with dash held on the remote runway.
+* _maxTiles_: The maximum horizontal speed that will satisfy the condition, measured in effective runway tiles with dash held on the remote runway.
+
+A `comeInSpaceJumping` entrance condition must match with a `leaveSpaceJumping` on the other side of the door. To match, the `blue` property of `leaveSpaceJumping` must be "no" or "any". This comes with an implicit `SpaceJump` item requirement.
+
+```json
+{
+  "name": "Come In Space Jumping",
+  "notable": false,
+  "entranceCondtion": {
+    "comeInSpaceJumping": {
+      "speedBooster": "any",
+      "minTiles": 4
+    }
+  },
+  "requires": [
+    "canCrossRoomJumpIntoWater"
```

Comment:
Yeah it's intended to be similar to comeInJumping. The idea is that with careful timing you can Space Jump at the "bottom" of the doorway; it's just that this "bottom" is a bit higher than a regular jump in the doorway. If a particular setup didn't allow a possibility of Space Jumping from the bottom of the door, then we wouldn't put "leaveSpaceJumping" on it; but it could still get a "leaveSpinning" using Space Jump.